### DISCUSSION TITLE
Support for fluid typography

### DISCUSSION THREAD

**Discussion by drawcard** [elementor/elementor#20858] [Feature Request]
Wordpress now has full support for fluid typography in Gutenberg: https://make.wordpress.org/core/2022/10/03/fluid-font-sizes-in-wordpress-6-1/

Similarly it would be wonderful to see fluid typography support in Elementor as well. The main benefit would be that it helps users quickly set a size in their typography that magically scales up and down depending on viewport width.

Currently you can achieve a similar thing if you write your own CSS to override Elementor text sizes, but it's not ideal and has a sharp learning curve.

**Comment by drawcard** [comment]
Actually I just read there's now support for clamp() and calc() in Elementor 3.10 - this is very helpful but still it requires user knowledge of what those functions mean. There could still be an opportunity to have a very simple text sizing range along the lines of:

![image](https://user-images.githubusercontent.com/1435638/209605665-0b7ae76e-4de3-4b9e-90d6-dbc85c739a23.png)

I don't recommend replacing the existing sliders & capabilities already in Elementor for font sizing, but maybe there could be a toggle or preference somewhere to "Enable simple typography" which shows something along the lines of the screenshot above?

**Reply by salpilk** [reply]
clamp() sizing is not working for me at all in a fresh install of elementor on a new site I'm working on. I have used clamp() successfully on other elementor sites. I am using the new flex container layout on my new site - maybe that is stopping the clamp settings from taking effect? Elementor support say they can't help with this as it is 'custom code'.  I am putting the clamp values into the Site settings >> Global Fonts settings.
I would much rather have the option of using clamp than having 5 or 6 different custom break point to input text sizes for.

**Reply by artkrsk** [reply]
For those who are still struggling with clamp formulas for the fluid typography: [Fluid Design System for Elementor](https://wordpress.org/plugins/fluid-design-system-for-elementor/)
